packages/react/src/checkbox/index.ts

```diff
@@ -1 +1,3 @@
 export * as Checkbox from './index.parts';
+
+// export { CheckboxRootProps } from './root/CheckboxRoot';
```

Comment:
This seems to match this fix:

https://github.com/typescript-eslint/typescript-eslint/pull/7690/files

Consumers can still use `Checkbox.Root.Props` on their wrapped components without issues, right?

Reply:
Yes, that's essentially what this PR demonstrates:

* define the interfaces top-level
* re-export them from the namespace
* re-export them from a public module (and namespace them in the export name to create base-wide unique name)

consumers can choose to either use the namespaced one or the top-level one, it would be the same.

It does seem to have a slight effect on the API docs though looking at CI

Reply:
btw, I tried the other way around

```tsx
export type CheckboxRootProps = Checkbox.Root.Props;
```

but that doesn't seem to work